rbd/rbd_nautilus.go

```diff
@@ -101,3 +101,17 @@ func (image *Image) GetModifyTimestamp() (Timespec, error) {
 
 	return Timespec(ts.CStructToTimespec(ts.CTimespecPtr(&cts))), nil
 }
+
+// Sparsify makes an image sparse by deallocating runs of zeros.
+// The sparseSize value will be used to find runs of zeros and must be
+// a power of two no less than 4096 and no larger than the image size.
```

Comment:
@dillaman Can you comment on this description? I wonder if 4096 is enforced, or is that just the size of the RADOS objects/stripes or some other configurable value.

Reply:
https://github.com/ceph/ceph/blob/master/src/librbd/Operations.cc#L1675